src/components/Form/FormFields/Autocomplete.tsx

```diff
@@ -74,6 +77,7 @@ type AutocompleteProps<T, V = T> = {
   placeholder?: string;
   optionLabel: OptionCallback<T, string>;
   optionIcon?: OptionCallback<T, React.ReactNode>;
+  optionImage?: OptionCallback<T | UserModel, string>;
```

Comment:
`Autocomplete` a reusable component has nothing to do with `User` in specific right?

```suggestion
  optionImage?: OptionCallback<T, string | undefined>;
```